editor/src/components/canvas/controls/select-mode/control-common.tsx

```diff
@@ -0,0 +1,116 @@
+import React from 'react'
```

Comment:
nit: the filename should probably be `controls-common`?

Reply:
done!